src/scaffoldmaker/meshtypes/meshtype_3d_spinalnerve1.py

```diff
@@ -0,0 +1,705 @@
+"""
+Generates a 3D spinal nerve scaffold using tube network mesh.
+"""
+from cmlibs.maths.vectorops import cross, mult, set_magnitude, magnitude, sub, euler_to_rotation_matrix, \
+    matrix_vector_mult, add
+from cmlibs.utils.zinc.field import Field, find_or_create_field_coordinates
+from cmlibs.zinc.node import Node
+from scaffoldmaker.annotation.annotationgroup import AnnotationGroup
+from scaffoldmaker.annotation.spinal_nerve_terms import get_spinal_nerve_term
+from scaffoldmaker.meshtypes.meshtype_1d_network_layout1 import MeshType_1d_network_layout1
+from scaffoldmaker.meshtypes.scaffold_base import Scaffold_base
+from scaffoldmaker.scaffoldpackage import ScaffoldPackage
+from scaffoldmaker.utils.interpolation import sampleCubicHermiteCurves, smoothCubicHermiteDerivativesLine
+from scaffoldmaker.utils.networkmesh import NetworkMesh
+from scaffoldmaker.utils.tubenetworkmesh import BodyTubeNetworkMeshBuilder, TubeNetworkMeshGenerateData
```

Comment:
Groups dorsal/ventral/left/right aren't correct.
Could remove; I think you only need to derive from TubeNetworkMeshBuilder to do this.
However, when you have the human pair or the whole spine it would be nice to label left and right.
Not so sure dorsal-ventral are worth keeping in their current form as there are already meaningful groups with those names for the nerve.